docs/docs/lock-stores.mdx

```diff
@@ -32,13 +32,84 @@ address the same node when sending messages for a given conversation ID.
   To use the `InMemoryTrackerStore` no configuration is needed.
 
 
+## ConcurrentRedisLockStore
+
+
+* **Description**
+
+  `ConcurrentRedisLockStore` uses Redis as a persistence layer for instances of issued [tickets](https://rasa.com/docs/rasa/reference/rasa/core/lock/#ticket-objects) and the last issued ticket number.
+  This Redis implementation of the `LockStore` can handle messages received in parallel for the same conversation by different Rasa servers.
+  Therefore, `ConcurrentRedisLockStore` ensures that only one Rasa instance can handle a conversation at any point in time.
+  This is the recommended lock store for running a replicated set of Rasa servers.
```

Comment:
```suggestion
  The `ConcurrentRedisLockStore` uses Redis as a persistence layer for instances of issued [tickets](https://rasa.com/docs/rasa/reference/rasa/core/lock/#ticket-objects) and the last issued ticket number.
  The `ConcurrentRedisLockStore` ensures that only one Rasa instance can handle a conversation at any point in time.
    Therefore this Redis implementation of the `LockStore` can handle messages received in parallel for the same conversation by different Rasa servers.
  This is the recommended lock store for running a replicated set of Rasa servers.
```

Reply:
I would suggest specifying the inheritance from the regular `RedisLockStore` and describing the improvements over it up front